editor/resources/console-view.fxml

```diff
@@ -13,7 +13,9 @@
     <RowConstraints vgrow="NEVER" />
   </rowConstraints>
   <HBox id="console-tool-bar" GridPane.rowIndex="0">
-    <Button id="filter-console" text="Filter" />
+    <Button id="filter-console" text="Filters" />
```

Comment:
Now that we have a toggle button next to the dropdown, I think it makes sense to use a plural noun for the multiselect filter list.